src/screens/MessageScreen.tsx

```diff
@@ -276,6 +310,16 @@ const defaultStyles = createStyles('MessageScreen', (theme) => {
       color: theme.colors.text,
       fontWeight: '600',
     },
+    deleteButtonView: {
+      backgroundColor: theme.colors.error,
+      justifyContent: 'center',
+      alignItems: 'flex-end',
+    },
+    deleteButtonLabel: {
```

Comment:
The color should probably be `theme.colors.onError`

Reply:
Looks like I have a conflict, I'll grab this during the rebase.